cogboard-webapp/src/utils/components.js

```diff
@@ -25,5 +25,8 @@ export const hasError = error => error !== undefined;
 export const getWidgetStatus = content =>
   (content && content.widgetStatus) || 'UNDEFINED';
 
-export const getWidgetUpdateTime = content =>
-  (content && content.lastUpdated) || 'UNDEFINED';
+export const getWidgetUpdateTime = (content, widgetType) => {
+  return widgetType.showUpdateTime
+    ? (content && content.lastUpdated) || 'UNDEFINED'
+    : false;
```

Comment:
I think you don't need ternary here and short evaluation expression will do the job, because you expect false when widgetType.showUpdateTime is falsy.

`widgetType.showUpdateTime && ((content && content.lastUpdated) || 'UNDEFINED')`

Should work the same, but check it please.

Reply:
indeed , it does the job